testing/workflows/components/workflows.libsonnet

```diff
@@ -224,7 +208,7 @@
             "--skip_tests=test_simple_tfjob_gpu",
           ],
         },  // run tests
-        dependencies: ["wait-for-kubeflow"],
+        dependencies: null,
```

Comment:
Do we need to wait kubeflow to be ready?

Reply:
I've just modified kfTests which is used as a sub workflow in kfctl-go-test (initially it was used at kfctl-test workflow). kfctl-go-test does not have the "wait-for"kubeflow" step. instead it has kfctl-is-ready step (see http://testing-argo.kubeflow.org/workflows/kubeflow-test-infra/kubeflow-presubmit-kfctl-go-iap-istio-3338-7797fce-8546-8147?tab=workflow)

Reply:
Should we depend on " kfctl-is-ready"?